allennlp/interpret/influence_interpreters/influence_interpreter.py

```diff
@@ -0,0 +1,361 @@
+from os import PathLike
+import re
+from typing import List, Optional, NamedTuple, Sequence, Union, Dict, Any
+
+import torch
+from torch import autograd
+
+from allennlp.common import Registrable, Lazy, plugins
+from allennlp.common.tqdm import Tqdm
+from allennlp.common.util import int_to_device
+from allennlp.data import Instance, DatasetReader, DatasetReaderInput, Batch
+from allennlp.data.data_loaders import DataLoader, SimpleDataLoader
+from allennlp.models import Model, Archive, load_archive
+from allennlp.nn.util import move_to_device
+
+
+class TrainInstanceInfluence(NamedTuple):
+    instance: Instance
+
+    loss: float
+
+    score: float
+    """
+    The influence score associated with this training instance.
+    """
+
+
+class InterpretOutput(NamedTuple):
+    test_instance: Instance
+
+    loss: float
+    """
+    The loss corresponding to the `test_instance`.
+    """
+
+    top_k: List[TrainInstanceInfluence]
+    """
+    The top `k` most influential training instances along with their influence score.
+    """
+
+    bottom_k: List[TrainInstanceInfluence]
```

Comment:
I would expect the bottom k to be basically noise.  Not that it's a problem to have this, but you might want to at least warn people in the docstring that this is only questionably useful for most methods.

Reply:
Yea, that's a good point. I don't see the value of keeping it, so I'm going to remove.